script/dev-deploy

```diff
@@ -863,6 +906,17 @@ class PrometheusService(Service):
             if config_path
             else ""
         )
+
+        resource_constraints = (
+            f"--memory={self.prometheus_memory_cpus[0]}"
+            if self.prometheus_memory_cpus[0] is not None
+            else ""
+        ) + (
+            f" --cpus={self.prometheus_memory_cpus[1]}"
+            if self.prometheus_memory_cpus[1] is not None
+            else ""
+        )
+
```

Comment:
maybe add a helper function to avoid duplicated? sth like:
``` Python
set_resource_constraints(memory, cpu):
  mem_constraints = memory if memory else ""
  cpu_constraints = cpu if cpu else ""
  return ( f" --memory={mem_constraints} --cpus={cpu_constraints}" )

opts += set_resource_constraints(...)
```
Also, I think no need to separate resource_constraints from opts. If you're interested, you can abstract all the parts related to configuration operations into different functions:
``` Python
opts = remove_when_done(rm)
opts += set_config_path(config_path)
opts += set_resource_constraints(...)
...
```

Reply:
that's neater, i gonna file a fix

Reply:
fixed. thx a lot